components/VC/common/VCCardSkeleton.tsx

```diff
@@ -4,9 +4,52 @@ import {ImageBackground, View} from 'react-native';
 import React from 'react';
 import LinearGradient from 'react-native-linear-gradient';
 import ShimmerPlaceholder from 'react-native-shimmer-placeholder';
+import {VCItemContainerFlowType, VCShareFlowType} from '../../../shared/Utils';
 
-export const VCCardSkeleton = () => {
-  return (
+export const VCCardSkeleton: React.FC<VCCardSkeletonProps> = props => {
+  return props.flow === VCItemContainerFlowType.OPENID4VP ? (
```

Comment:
Let's check the design once, can we have same Skeleton for both the flows

Reply:
Swati this would require discussion with sanchi so can we take this up as part of another card as UI enhancement?